code/modules/mob/living/carbon/xenomorph/Embryo.dm

```diff
@@ -111,15 +114,19 @@
 		if(4)
 			if(prob(1))
 				if(affected_mob.knocked_out < 1)
+					affected_mob.pain.apply_pain(PAIN_BONE_BREAK)
 					affected_mob.visible_message(SPAN_DANGER("\The [affected_mob] starts shaking uncontrollably!"), \
 												 SPAN_DANGER("You start shaking uncontrollably!"))
 					affected_mob.apply_effect(10, PARALYZE)
 					affected_mob.make_jittery(105)
 					affected_mob.take_limb_damage(1)
 			if(prob(2))
+				affected_mob.pain.apply_pain(PAIN_BONE_BREAK)
 				var/message = pick("Your chest hurts badly", "It becomes difficult to breathe", "Your heart starts beating rapidly, and each beat is painful")
```

Comment:
these messages are kind of underwhelming compared to the pain this stage is apparently putting them through

Reply:
the `apply_pain` brings them up from nothing to mild, and not even always that (pain code). It stays in these states until the larva gets occupied and chestbursting begins which brings the pain on for real. 
![image](https://user-images.githubusercontent.com/16618648/202831582-ba27ccce-7772-47a9-81da-4ac30b7fcc1e.png)